UnityProject/Assets/Scripts/Core/Chat/ChatEntry.cs

```diff
@@ -172,20 +173,34 @@ public void SetText(string message, TMP_SpriteAsset languageSprite, TMP_FontAsse
 
 		public void AddChatDuplication()
 		{
-			if (stackCount == 1)
-			{
-				// Just need to do this once; message size won't change.
-				SetStackPos();
-			}
-
+			SetStackPos(); // Always make sure the stack bubble has its position updated incase the entry gets updated.
 			stackText.text = $"x{++stackCount}";
 			ToggleUIElements(true);
 			stackObject.SetActive(true);
 			AnimateFade(1f, 0f);
 			StartCoroutine(AnimateStackObject());
+			StackMessageSizeIncrease();
 			this.RestartCoroutine(FadeCooldown(), ref fadeCooldownCoroutine);
 		}
 
+		private void StackMessageSizeIncrease()
+		{
+			string pattern = @"<size(=|\+=|\+)([0-9]+|[0-9]+\+|=[0-9]+\+)>";
```

Comment:
what's with the + at the end? Can the number be something like ``10+``?

Reply:
yes, it can be like this:

<size=[number]+>
<size=+[number]>
<size+=[number]>

Reply:
what are all the possible cases? Could it ever be ``<size+10`` for example?

Reply:
...I just posted all the possible cases

Reply:
![image](https://github.com/unitystation/unitystation/assets/43683714/12318604-c353-4687-995b-4d6fe2e93890)

your second case doesn't match and it matches a case that shouldn't be valid

Reply:
what characters do you want to capture from the matches? I'm guessing the symbol and the value, right?

Reply:
![image](https://github.com/unitystation/unitystation/assets/43683714/28adc23e-9d3e-4afb-a270-d6468e8c2e48)
this one should be enough

``<size(=|\+=|=+)(\+?[0-9]+\+?)>``

Reply:
I pushed a commit that should resolve this:
`<Size(=|\+?=)?([0-9]+|\+[0-9]+|[0-9]+\+)?>`

Reply:
``<size(=|\+=)(\+?[0-9]+\+?)>``

I had one unused alternative inside the first case

Reply:
nevermind, it broke completely ingame

Reply:
> `<size(=|\+=)(\+?[0-9]+\+?)>`
> 
> I had one unused alternative inside the first case

![image](https://github.com/unitystation/unitystation/assets/34368774/1b44ef0e-b523-4688-a49d-9e4f954d8eed)

this works, but now running into this funny issue

Reply:
removed it, but problem still presists